docs/toc.yml

```
@@ -122,7 +122,7 @@
         displayName: event-based activation
       - name: Run code on document open
         href: develop/run-code-on-document-open.md
-    - name: Manifest
+    - name: Manifest and app package
```

```suggestion
    - name: App package and manifest
```
If you accept the suggestion to reorder this section.